packages/contracts/package.json

```diff
@@ -57,6 +57,7 @@
         "ethers-typescript-typings": "^0.0.4",
         "mocha": "^4.0.1",
         "npm-run-all": "^4.1.2",
+        "prettier": "^1.11.1",
```

Comment:
Remove that

Reply:
`prettier --write` is part of the `generate_contract_wrappers` command in `contracts` repo. We need `prettier` as a devDep.